### ISSUE TITLE
Add Link to eBook in Node-RED page

### ISSUE THREAD

**Issue by Hasmin-AC** [FlowFuse/website#1791]
### Description

This can be added as a link in the "Learn Node-RED" section or as its own call-out box with a graphic just under it. 

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by Yndira-E** [comment]
@Hasmin-AC I'll add it to the key concepts column, I don't think that document qualifies as a quick tip 😄

**Comment by Yndira-E** [comment]
Do you want it in any particular order? First? Last?

**Comment by Hasmin-AC** [comment]
Yes agree its not a quick tip and neither is it a concept, so can we make it span across the two columns as a learning resource by itself?

**Comment by Yndira-E** [comment]
It would be like this

![Image](https://github.com/FlowFuse/website/assets/129537638/d57c282e-0961-429c-af7b-8e3fd90a3057)

But I think the space to the right is too much, how about like this? And seems more obvious that it's not under quick tips or key concepts (at least on desktop)


![Image](https://github.com/FlowFuse/website/assets/129537638/4d1cfd7a-4002-4824-ac3f-120ef53cc3d7)
